generate_x509_key_and_certificate.sh

```diff
@@ -0,0 +1,31 @@
+CERTIFICATES_DIR="`pwd`/example/certificates/"
```

Comment:
Qui dovremmo usare una strategia simile a questo piuttosto:
https://github.com/italia/Satosa-Saml2Spid/blob/master/build_spid_certs.sh

Reply:
anche qui non ho capito bene che cosa dovrei fare... video call? 😅

Reply:
i certificati li devi creare secondo le specifiche AgID, dai uno sguardo a quello script scritto da paolo smiraglia
è solo un esempio ma si avvicina molto di più alla desiderata

Reply:
Ecco, su questo punto avevo cominciato su Django-Identity (https://github.com/Martini991/Django-Identity/tree/docker/generate-spid-certificates) ed avevo preso spunto dallo script che mi hai indicato, solo che non sono super-esperto in materia ed avrei bisogno di un aiuto per verificare se in quel modo i certificati sono conformi alle specifiche AgID.

Reply:
non sappiamo se AgID abbia cambiato procedure ma di certo questo script è un'ottima base.
Possiamo informarci ma per adesso tendiamo a produrre gli script con il tool di psmiraglia

Reply:
Dovremmo creare un repository apposta per generare i certificati, magari qualcosa come "italia/spid-certificates" e poi lavorare su quello; gli altri repository poi si appoggerebbero a quello. Addirittura si potrebbe anche creare un immagine Docker per generare al volo i certificati.

Reply:
su questo conto di lavorarci a breve, devo fare "un giro" prima di finalizzare